Gruntfile.js

```diff
@@ -38,12 +38,27 @@ module.exports = function(grunt) {
 
   grunt.registerTask('build', ['clean:dist', 'sass', 'webpack']);
 
+  // The dev server used for live reloading in the browser.
   grunt.registerTask('serve',
     ['clean:dist', 'sass:dev', 'webpack-dev-server:start', 'watch:sass']);
-  grunt.registerTask('start',
-    ['clean:dist', 'sass:dev', 'webpack:dev', 'watch:sass']);
-  grunt.registerTask('start-email',
+  // Used to serve email CSS.
+  grunt.registerTask('watch-email',
     ['clean:dist', 'sass:email', 'webpack:dev', 'watch:sassemail']);
+  // Regular grunt server, used when running inside the payments-env docker.
+  grunt.registerTask('watch-static',
+    ['clean:dist', 'sass:dev', 'webpack:dev', 'watch:sass']);
+  // Legacy tasks still used by payments-env, see:
+  // https://github.com/mozilla/payments-ui/issues/361
+  grunt.registerTask('start', function() {
+    grunt.log.writeln(
+      '"grunt start" is deprecated; use `grunt watch-static` instead');
+    grunt.task.run('serve');
```

Comment:
When using funcs in registerTask the middle arg between the taskname  + func can be a description output by --help. See http://gruntjs.com/api/grunt.task#grunt.task.registertask for more details. That might be nice for showing what's deprecated.